src/App/Home/Home.js

```diff
@@ -2,15 +2,25 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { fetchRavenPayments } from './actions';
 import { WorkerStats } from '../WorkerStats/WorkerStats';
+import { NavBar } from '../NavBar/NavBar';
+import { WorkerInput } from '../WorkerInput/WorkerInput';
 
 class Home extends React.Component {
     componentWillMount(){
         this.props.onFetchRavenPayments('123');
     }
     render() {
+        const showWorkerStats = true;
+        const showWorkerInput = false;
         return (
             <main>
-                <WorkerStats />
+                <NavBar />
+                {showWorkerInput &&

```

Comment:
Would you be opposed to moving the showWorkerInput and showWorkerStats into their components?

Reply:
Yup, working on it now. :)